find.go

```diff
@@ -15,29 +15,57 @@ import (
 func (s *server) find(w http.ResponseWriter, r *http.Request) {
 	combined := make(chan *model.FindResponse, len(s.servers))
 	wg := sync.WaitGroup{}
+
+	// Copy the original request body in case it is a POST batch find request.
+	var rb []byte
+	var err error
+	rb, err = io.ReadAll(r.Body)
```

Comment:
do we need to limit our buffer somehow here? hopefully cloudfront would put a request body size limit in front of this already

Reply:
Limited to `8 KiB`.